ocaml/xapi-consts/api_errors.ml

```diff
@@ -550,6 +550,16 @@ let crl_does_not_exist = "CRL_DOES_NOT_EXIST"
 let crl_already_exists = "CRL_ALREADY_EXISTS"
 let crl_name_invalid = "CRL_NAME_INVALID"
 let crl_corrupt = "CRL_CORRUPT"
+let server_certificate_key_invalid = "SERVER_CERTIFICATE_KEY_INVALID"
```

Comment:
Don't forget to define these errors with their doc strings and parameters in `datamodel_errors.ml`.